cobra/util/version_info.py

```diff
@@ -55,7 +55,8 @@ def get_pkg_info():
     # using requirements files that can be read in.
     dependencies = frozenset(PKG_ORDER)
     blob = dict()
-    for dist in pip.get_installed_distributions():
+    dists = [d for d in pkg_resources.working_set]
```

Comment:
Since `dists` is not used for anything else can you combine these two lines to:

```python
for dist in pkg_resources.working_set:
```
please?